widget/ui/src/components/Divider/Divider.tsx

```diff
@@ -9,6 +9,14 @@ const DEFAULT_SIZE = 12;
 export function Divider({
   size = DEFAULT_SIZE,
   direction = 'vertical',
+  color,
 }: PropTypes) {
-  return <DividerContainer size={size} direction={direction} />;
+  const customCss = color
+    ? {
+        backgroundColor: color.startsWith('$') ? color : `$${color}`,
+      }
+    : {
+        backgroundColor: 'transparent',
+      };
+  return <DividerContainer size={size} direction={direction} css={customCss} />;
```

Comment:
isn't possible to change it to a prop for css or style instead? current code is not very clean.

Reply:
I wanted to have a similar style to the Typography component. however, changed.